src/vcpkg/vcpkgpaths.cpp

```diff
@@ -597,10 +597,10 @@ namespace vcpkg
 
     DECLARE_AND_REGISTER_MESSAGE(
         ErrorMissingVcpkgRoot,
-        (msg::url),
+        (),
```

Comment:
I had extracted out the url since it isn't localizable content.

Reply:
I think it's OK to for messages to have some content which isn't localizable; for example we talk about command line flags and stuff that aren't localized either. The comment needs to describe this. (Putting it back though)

Reply:
Actually, not putting it back because I changed these instructions to ask people to file GitHub issues.